### PR TITLE
Fixes - User permissions does not work in listview in 7.10.4

### PR DESCRIPTION
**Pull request by RobertCopilau**
http://issues.umbraco.org/issue/UAAS-745#

### CODE REVIEW

**Comment by poornimanayar** [comment]
Hi,

Many thanks for your PR. We shall review your PR and let you know of the outcome within the next 14 days.

Regards
Poornima

File: src/Umbraco.Web/Trees/ContentTreeControllerBase.cs [hunk 1]
```diff
@@ -203,7 +203,9 @@ protected IEnumerable<IUmbracoEntity> GetChildEntities(string id)
                 entityId = entity.Id;
             }
 
-            return Services.EntityService.GetChildren(entityId, UmbracoObjectType).ToArray();
+            var v2 = Services.EntityService.GetChildren(entityId, UmbracoObjectType).ToList();
```
Comment (poornimanayar) [inline src/Umbraco.Web/Trees/ContentTreeControllerBase.cs]: Hi, can we rename the variable to something more meaningful?

**Comment by poornimanayar** [comment]
Hi @RobertCopilau ,

I have tested your changes as per the original issue. I have tested for list items, child items as well as random nodes just to make sure everything is intact. Please do let me know if there is any other test case I need to test for. If you can address my comment and commit to the PR again that will be awesome.

Regards,
Poornima([The Community Pull Request Team](https://umbraco.com/blog/introducing-the-new-pr-team/))

File: src/Umbraco.Web/Trees/ContentTreeControllerBase.cs (hunk 1 above)
Comment (RobertCopilau) [reply src/Umbraco.Web/Trees/ContentTreeControllerBase.cs]: Haha yes, my bad, just a leftover debug artifact.
Thanks for letting me know :D.

**Pull request merged**